
<!DOCTYPE html>

<html class="no-js" lang="en">
<head>
<meta charset="utf-8"/>
<meta content="width=device-width,initial-scale=1" name="viewport"/>
<link href="https://sepio-framework.github.io/sepio-linkml/StudyResult/" rel="canonical"/>
<link href="../assets/images/favicon.png" rel="icon"/>
<meta content="mkdocs-1.6.1, mkdocs-material-8.5.10" name="generator"/>
<title>Class: Study Result (StudyResult) - sepio-linkml</title>
<link href="../assets/stylesheets/main.975780f9.min.css" rel="stylesheet"/>
<link href="../assets/stylesheets/palette.2505c338.min.css" rel="stylesheet"/>
<link crossorigin="" href="https://fonts.gstatic.com" rel="preconnect"/>
<link href="https://fonts.googleapis.com/css?family=Roboto:300,300i,400,400i,700,700i%7CRoboto+Mono:400,400i,700,700i&amp;display=fallback" rel="stylesheet"/>
<style>:root{--md-text-font:"Roboto";--md-code-font:"Roboto Mono"}</style>
<script>__md_scope=new URL("..",location),__md_hash=e=>[...e].reduce((e,_)=>(e<<5)-e+_.charCodeAt(0),0),__md_get=(e,_=localStorage,t=__md_scope)=>JSON.parse(_.getItem(t.pathname+"."+e)),__md_set=(e,_,t=localStorage,a=__md_scope)=>{try{t.setItem(a.pathname+"."+e,JSON.stringify(_))}catch(e){}}</script>
</head>
<body data-md-color-accent="None" data-md-color-primary="None" data-md-color-scheme="default" dir="ltr">
<input autocomplete="off" class="md-toggle" data-md-toggle="drawer" id="__drawer" type="checkbox"/>
<input autocomplete="off" class="md-toggle" data-md-toggle="search" id="__search" type="checkbox"/>
<label class="md-overlay" for="__drawer"></label>
<div data-md-component="skip">
<a class="md-skip" href="#class-study-result-studyresult">
          Skip to content
        </a>
</div>
<div data-md-component="announce">
</div>
<header class="md-header" data-md-component="header">
<nav aria-label="Header" class="md-header__inner md-grid">
<a aria-label="sepio-linkml" class="md-header__button md-logo" data-md-component="logo" href=".." title="sepio-linkml">
<svg viewbox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><path d="M12 8a3 3 0 0 0 3-3 3 3 0 0 0-3-3 3 3 0 0 0-3 3 3 3 0 0 0 3 3m0 3.54C9.64 9.35 6.5 8 3 8v11c3.5 0 6.64 1.35 9 3.54 2.36-2.19 5.5-3.54 9-3.54V8c-3.5 0-6.64 1.35-9 3.54Z"></path></svg>
</a>
<label class="md-header__button md-icon" for="__drawer">
<svg viewbox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><path d="M3 6h18v2H3V6m0 5h18v2H3v-2m0 5h18v2H3v-2Z"></path></svg>
</label>
<div class="md-header__title" data-md-component="header-title">
<div class="md-header__ellipsis">
<div class="md-header__topic">
<span class="md-ellipsis">
            sepio-linkml
          </span>
</div>
<div class="md-header__topic" data-md-component="header-topic">
<span class="md-ellipsis">
            
              Class: Study Result (StudyResult)
            
          </span>
</div>
</div>
</div>
<label class="md-header__button md-icon" for="__search">
<svg viewbox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"></path></svg>
</label>
<div class="md-search" data-md-component="search" role="dialog">
<label class="md-search__overlay" for="__search"></label>
<div class="md-search__inner" role="search">
<form class="md-search__form" name="search">
<input aria-label="Search" autocapitalize="off" autocomplete="off" autocorrect="off" class="md-search__input" data-md-component="search-query" name="query" placeholder="Search" required="" spellcheck="false" type="text"/>
<label class="md-search__icon md-icon" for="__search">
<svg viewbox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"></path></svg>
<svg viewbox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11h12Z"></path></svg>
</label>
<nav aria-label="Search" class="md-search__options">
<button aria-label="Clear" class="md-search__icon md-icon" tabindex="-1" title="Clear" type="reset">
<svg viewbox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><path d="M19 6.41 17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12 19 6.41Z"></path></svg>
</button>
</nav>
</form>
<div class="md-search__output">
<div class="md-search__scrollwrap" data-md-scrollfix="">
<div class="md-search-result" data-md-component="search-result">
<div class="md-search-result__meta">
            Initializing search
          </div>
<ol class="md-search-result__list"></ol>
</div>
</div>
</div>
</div>
</div>
<div class="md-header__source">
<a class="md-source" data-md-component="source" href="https://github.com/sepio-framework/sepio-linkml" title="Go to repository">
<div class="md-source__icon md-icon">
<svg viewbox="0 0 448 512" xmlns="http://www.w3.org/2000/svg"><!--! Font Awesome Free 6.2.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2022 Fonticons, Inc.--><path d="M439.55 236.05 244 40.45a28.87 28.87 0 0 0-40.81 0l-40.66 40.63 51.52 51.52c27.06-9.14 52.68 16.77 43.39 43.68l49.66 49.66c34.23-11.8 61.18 31 35.47 56.69-26.49 26.49-70.21-2.87-56-37.34L240.22 199v121.85c25.3 12.54 22.26 41.85 9.08 55a34.34 34.34 0 0 1-48.55 0c-17.57-17.6-11.07-46.91 11.25-56v-123c-20.8-8.51-24.6-30.74-18.64-45L142.57 101 8.45 235.14a28.86 28.86 0 0 0 0 40.81l195.61 195.6a28.86 28.86 0 0 0 40.8 0l194.69-194.69a28.86 28.86 0 0 0 0-40.81z"></path></svg>
</div>
<div class="md-source__repository">
    GitHub
  </div>
</a>
</div>
</nav>
</header>
<div class="md-container" data-md-component="container">
<main class="md-main" data-md-component="main">
<div class="md-main__inner md-grid">
<div class="md-sidebar md-sidebar--primary" data-md-component="sidebar" data-md-type="navigation">
<div class="md-sidebar__scrollwrap">
<div class="md-sidebar__inner">
<nav aria-label="Navigation" class="md-nav md-nav--primary" data-md-level="0">
<label class="md-nav__title" for="__drawer">
<a aria-label="sepio-linkml" class="md-nav__button md-logo" data-md-component="logo" href=".." title="sepio-linkml">
<svg viewbox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><path d="M12 8a3 3 0 0 0 3-3 3 3 0 0 0-3-3 3 3 0 0 0-3 3 3 3 0 0 0 3 3m0 3.54C9.64 9.35 6.5 8 3 8v11c3.5 0 6.64 1.35 9 3.54 2.36-2.19 5.5-3.54 9-3.54V8c-3.5 0-6.64 1.35-9 3.54Z"></path></svg>
</a>
    sepio-linkml
  </label>
<div class="md-nav__source">
<a class="md-source" data-md-component="source" href="https://github.com/sepio-framework/sepio-linkml" title="Go to repository">
<div class="md-source__icon md-icon">
<svg viewbox="0 0 448 512" xmlns="http://www.w3.org/2000/svg"><!--! Font Awesome Free 6.2.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2022 Fonticons, Inc.--><path d="M439.55 236.05 244 40.45a28.87 28.87 0 0 0-40.81 0l-40.66 40.63 51.52 51.52c27.06-9.14 52.68 16.77 43.39 43.68l49.66 49.66c34.23-11.8 61.18 31 35.47 56.69-26.49 26.49-70.21-2.87-56-37.34L240.22 199v121.85c25.3 12.54 22.26 41.85 9.08 55a34.34 34.34 0 0 1-48.55 0c-17.57-17.6-11.07-46.91 11.25-56v-123c-20.8-8.51-24.6-30.74-18.64-45L142.57 101 8.45 235.14a28.86 28.86 0 0 0 0 40.81l195.61 195.6a28.86 28.86 0 0 0 40.8 0l194.69-194.69a28.86 28.86 0 0 0 0-40.81z"></path></svg>
</div>
<div class="md-source__repository">
    GitHub
  </div>
</a>
</div>
<ul class="md-nav__list" data-md-scrollfix="">
<li class="md-nav__item">
<a class="md-nav__link" href="..">
        Index
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../about/">
        About
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../core-im-diagram/">
        Core IM Diagram
      </a>
</li>
</ul>
</nav>
</div>
</div>
</div>
<div class="md-sidebar md-sidebar--secondary" data-md-component="sidebar" data-md-type="toc">
<div class="md-sidebar__scrollwrap">
<div class="md-sidebar__inner">
<nav aria-label="Table of contents" class="md-nav md-nav--secondary">
<label class="md-nav__title" for="__toc">
<span class="md-nav__icon md-icon"></span>
      Table of contents
    </label>
<ul class="md-nav__list" data-md-component="toc" data-md-scrollfix="">
<li class="md-nav__item">
<a class="md-nav__link" href="#comments">
    Comments
  </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="#inheritance">
    Inheritance
  </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="#slots">
    Slots
  </a>
<nav aria-label="Slots" class="md-nav">
<ul class="md-nav__list">
<li class="md-nav__item">
<a class="md-nav__link" href="#direct-slots">
    Direct slots
  </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="#inherited-slots">
    Inherited slots
  </a>
</li>
</ul>
</nav>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="#usages">
    Usages
  </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="#identifier-and-mapping-information">
    Identifier and Mapping Information
  </a>
<nav aria-label="Identifier and Mapping Information" class="md-nav">
<ul class="md-nav__list">
<li class="md-nav__item">
<a class="md-nav__link" href="#schema-source">
    Schema Source
  </a>
</li>
</ul>
</nav>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="#mappings">
    Mappings
  </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="#linkml-source">
    LinkML Source
  </a>
<nav aria-label="LinkML Source" class="md-nav">
<ul class="md-nav__list">
<li class="md-nav__item">
<a class="md-nav__link" href="#direct">
    Direct
  </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="#induced">
    Induced
  </a>
</li>
</ul>
</nav>
</li>
</ul>
</nav>
</div>
</div>
</div>
<div class="md-content" data-md-component="content">
<article class="md-content__inner md-typeset">
<a class="md-content__button md-icon" href="https://github.com/sepio-framework/sepio-linkml/edit/master/docs/StudyResult.md" title="Edit this page">
<svg viewbox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><path d="M20.71 7.04c.39-.39.39-1.04 0-1.41l-2.34-2.34c-.37-.39-1.02-.39-1.41 0l-1.84 1.83 3.75 3.75M3 17.25V21h3.75L17.81 9.93l-3.75-3.75L3 17.25Z"></path></svg>
</a>
<h1 id="class-study-result-studyresult">Class: Study Result (StudyResult)</h1>
<p><em>A collection of data items from a single study that pertain to a particular subject or experimental unit in the study, along with optional provenance information describing how these data items were generated.</em></p>
<h2 id="comments">Comments</h2>
<ul>
<li>
<ul>
<li>StudyResults provide a useful way to capture a subset of items from a study dataset that are used as evidence in generating higher order knowledge assertions about the entity that is the focus of the study result. For example, a subset of allele frequency data items about a particular variant of interest in the complete gnomAD dataset can be reported in a StudyResult object, along with relevant provenance information, so they can be referenced as evidence for a higher-order assertion about the variant's pathogenicity.</li>
</ul>
</li>
<li>As an example, consider the comprehensive allele frequency dataset provided by gnomAD, which covers millions of variants. A curator looking to assess the pathogenicity of a particular variant might create a StudyResult object to capture a subset of data related to this focus allele, including  its count, frequency, homozygous frequency, along with metadata concerning the shared provenance or quality of this data. The StudyResult could then be references as a piece of evidence used to inform the focus alleles final pathogenicity classification.</li>
<li>Note that the use of a Study Results is not reserved only for defining subsets of data from a larger HTP/clinical study data sets.  A StudyResult might be used to organize the data from a simple, small scale bench experiment - e.g. a western blot analysis of protein expression, or an in vitro binding assay focused on a single protein.  Even such small 'studies' can generate multiple data items and metadata, and a StudyResult object can be used to collect all or some of these data points pertinent to a particular focus into an organized structure.</li>
</ul>
<p>URI: <a href="https://w3id.org/sepio-framework/sepio-linkml/StudyResult">sepio_linkml:StudyResult</a></p>
<div class="mermaid"> classDiagram
    class StudyResult
    click StudyResult href "../StudyResult"
      InformationEntity &lt;|-- StudyResult
        click InformationEntity href "../InformationEntity"

      StudyResult : alternativeLabels

      StudyResult : componentResult




    StudyResult --&gt; "*" StudyResult : componentResult
    click StudyResult href "../StudyResult"


      StudyResult : contributions




    StudyResult --&gt; "*" Contribution : contributions
    click Contribution href "../Contribution"


      StudyResult : dataItems




    StudyResult --&gt; "*" DataItem : dataItems
    click DataItem href "../DataItem"


      StudyResult : dateAuthored

      StudyResult : derivedFrom




    StudyResult --&gt; "*" InformationEntity : derivedFrom
    click InformationEntity href "../InformationEntity"


      StudyResult : description

      StudyResult : extensions




    StudyResult --&gt; "*" Extension : extensions
    click Extension href "../Extension"


      StudyResult : focus

      StudyResult : id

      StudyResult : identifiers

      StudyResult : informationQuality




    StudyResult --&gt; "0..1" Coding : informationQuality
    click Coding href "../Coding"


      StudyResult : interpretation




    StudyResult --&gt; "0..1" Coding : interpretation
    click Coding href "../Coding"


      StudyResult : isAbout

      StudyResult : label

      StudyResult : recordMetadata




    StudyResult --&gt; "0..1" RecordMetadata : recordMetadata
    click RecordMetadata href "../RecordMetadata"


      StudyResult : reportedIn

      StudyResult : sourceDataSet




    StudyResult --&gt; "*" DataSet : sourceDataSet
    click DataSet href "../DataSet"


      StudyResult : specifiedBy

      StudyResult : studyGroup




    StudyResult --&gt; "0..1" StudyGroup : studyGroup
    click StudyGroup href "../StudyGroup"


      StudyResult : supportingMethods

      StudyResult : supportingMethodTypes




    StudyResult --&gt; "*" Coding : supportingMethodTypes
    click Coding href "../Coding"


      StudyResult : type


</div>
<h2 id="inheritance">Inheritance</h2>
<ul>
<li><a href="../Entity/">Entity</a><ul>
<li><a href="../InformationEntity/">InformationEntity</a><ul>
<li><strong>StudyResult</strong></li>
</ul>
</li>
</ul>
</li>
</ul>
<h2 id="slots">Slots</h2>
<h3 id="direct-slots">Direct slots</h3>
<p><a href="../focus/">focus</a> --&gt; <a href="../String/">String</a> [0..1] <br/></p>
<blockquote>
<p>A specific subject or experimental unit in a Study, that data in the StudyResult object is about - e.g. a particular variant in a population allele frequency dataset like ExAC or gnomAD.</p>
</blockquote>
<details>
<summary>Implementation Guidance</summary>
* The 'focus' of a StudyResult is the what anchors selection of all data items and provenance information that it contains. This focus may be a single participant or subject of a study (e.g. one patient in a clinical study, or one allele in a population frequency analysis). It may be a study group defined as an experimental unit in the study (e.g. a specific treatment arm of a clinical study). Or it may be a variable defined as a unit of analysis in the study (e.g. 'exposure to nicotine' in an analysis correlating this variable with clinical outcomes).<br/>
</details>
<hr/>
<p><a href="../dataItems/">dataItems</a> --&gt; <a href="../DataItem/">DataItem</a> [*] <br/></p>
<blockquote>
<p>One or more data items that are included in the StudyResult because it pertains to the 'focus' of the result. This can be data that directly describes this 'focus' (e.g. the population frequency of an allele focus), or be metadata about how data about the 'focus' were generated (e.g the quality measures for the sequencing run used to determine this allele frequency).</p>
</blockquote>
<details>
<summary>Implementation Guidance</summary>
* Any type of data item from a Study may be captured in a StudyResult - including raw experimental/observational data, derived calculations or quality measures, provenance information about how the data were generated in the study, or qualitative expert interpretations of the primary study data (e.g. the interpretation of a blood cholesterol value of 250 mg/dL as being 'high', or a Polyphen functional impact score of 0.115 as being 'deleterious').<br/>
* The model specifies use of a key-value based 'DataItem' object to capture the meaning and value of each type of data item captured in a given StudyResult. But in practice, profiles for specific StudyResult types may choose to define one or more specializations of the generic 'dataItems' property as named attributes. This makes the data more succinct and parsable, and allows specific constraints to be applied and validated for different data items. For example, a PopulaitonAlleleFrequencyStudyResult profile may define a named 'focusAlleleFrequency' attribute that is required, and a named 'focusAlleleCount' attribute that is optional - both of which conceptually specialize the core-im 'dataItems' property. Under this approach, the core 'dataItems' acts as a placeholder to seed such specializations, but is not used directly in StudyResult profiles.<br>
</br></details>
<hr/>
<p><a href="../interpretation/">interpretation</a> --&gt; <a href="../Coding/">Coding</a> [0..1] <br/></p>
<blockquote>
<p>The outcome of an interpretation of lower-level data item(s) in a StudyResult, that express some broader conclusion or insight that was made (e.g. that data indicates a result to be is high, normal, or low).</p>
</blockquote>
<details>
<summary>Implementation Guidance</summary>
* This attribute allows basic interpretations / classifications to be included in a Study Result, such as binning a data item or items as being in or out of some normal range.<br/>
</details>
<hr/>
<p><a href="../sourceDataSet/">sourceDataSet</a> --&gt; <a href="../DataSet/">DataSet</a> [*] <br/></p>
<blockquote>
<p>A larger DataSet from which the content of the StudyResult was derived.</p>
</blockquote>
<details>
<summary>Implementation Guidance</summary>
* In most cases, a StudyResult will be generated using data from one source dataset - but it is possible multiple datasets related to a single study contain data about a particular focus that get collected into a single StudyResult instance.<br/>
</details>
<hr/>
<p><a href="../componentResult/">componentResult</a> --&gt; <a href="./">StudyResult</a> [*] <br/></p>
<blockquote>
<p>Another StudyResult comprised of data items about the same focus as its parent Result, but based on a more narrowly scoped analysis of the foundational data (e.g. an analysis based on data about a subset of the parent Results full study population).</p>
</blockquote>
<details>
<summary>Implementation Guidance</summary>
* This attribute allows data creators to break down a StudyResult into finer-grained StudyResult instances with a narrower scope. For example, a StudyResult about the frequency of an allele in a global population of individuals can be broken down into separate 'component' Results about distinct subpopulations of individuals in the source data (e.g. frequency of the allele in asian vs african subpopulations, or male vs female subgroups).<br/>
</details>
<hr/>
<p><a href="../studyGroup/">studyGroup</a> --&gt; <a href="../StudyGroup/">StudyGroup</a> [0..1] <br/></p>
<blockquote>
<p>A description of a specific group or population of subjects interrogated in the ResearchStudy that produced the data captured in the StudyResult.</p>
</blockquote>
<details>
<summary>Implementation Guidance</summary>
* For example, in a StudyResult holding allele frequency data, this attribute points to a StudyGroup object that describes characteristics of the population that the frequency data was generated from (e.g. 'East Asian', 'Female').<br/>
</details>
<hr/>
<h3 id="inherited-slots">Inherited slots</h3>
<p><a href="../isAbout/">isAbout</a> --&gt; <a href="../String/">String</a> [*] <br/></p>
<blockquote>
<p>An entity or concept in the world that the information entity describes/is about. <br/></p>
</blockquote>
<p>Inherited from: <a href="../InformationEntity/">InformationEntity</a></p>
<details>
<summary>Implementation Guidance</summary>
* e.g. In the context of a Statement object, this attribute may be used to indicate entities/concepts it is about, in lieu of providing a more precise description of what the Statement asserts to be true using subject, predicate, object, and qualifier properties. e.g. the Statement that "BRCA2 c.8023A&gt;G is pathogenic for Breast Cancer" might be annotated to be about the variant 'BRCA2 c.8023A&gt;G', and the disease 'Breast Cancer'.<br/>
</details>
<p><a href="../contributions/">contributions</a> --&gt; <a href="../Contribution/">Contribution</a> [*] <br/></p>
<blockquote>
<p>Specific actions taken by an Agent toward the creation, modification, validation, or deprecation of an Information Entity. <br/></p>
</blockquote>
<p>Inherited from: <a href="../InformationEntity/">InformationEntity</a></p>
<details>
<summary>Implementation Guidance</summary>
* This attribute holds one or more Contribution objects, which provide structured descriptions of a contribution made to the Information Entity by a particular agent.<br/>
</details>
<p><a href="../dateAuthored/">dateAuthored</a> --&gt; <a href="../String/">String</a> [0..1] <br/></p>
<blockquote>
<p>Indicates when the information content expressed in the Information Entity was generated. <br/></p>
</blockquote>
<p>Inherited from: <a href="../InformationEntity/">InformationEntity</a></p>
<details>
<summary>Implementation Guidance</summary>
* The term 'authored' as used in the model refers to the generation of 'information content' in the abstract  sense, as opposed to a concrete encoding of this information in a specific language or format. e.g. for a Statement, this attribute captures when the information content expressed in the Statement was first generated by an agent.  Information about when a particular concrete encoding of this information was created (e.g. as row in a table, or object in a json document) would live in a RecordMetadata object attached to the Information Entity).<br/>
</details>
<p><a href="../specifiedBy/">specifiedBy</a> --&gt; <a href="../String/">String</a> [*] <br/></p>
<blockquote>
<p>A specification that describes all or part of the process that led to creation of the Information Entity. <br/></p>
</blockquote>
<p>Inherited from: <a href="../InformationEntity/">InformationEntity</a></p>
<details>
<summary>Implementation Guidance</summary>
* Examples - an experimental protocol or data analysis specification that describe how data were generated, or an evidence interpretation guideline that describes steps taken to interpret data in making a variant pathogenicity classification.<br/>
* Note that this attribute captures specific *instances* of specifications/methods (e.g. the specific electron microscopy method described in https://doi.org/10.1002/ cpz1.1045) - as opposed to reporting a *type* of method applied (e.g. "Transmission Electron Microscopy").<br/>
</details>
<p><a href="../supportingMethods/">supportingMethods</a> --&gt; <a href="../String/">String</a> [*] <br/></p>
<blockquote>
<p>Specific methods that were executed to directly or indirectly support creation of the Information Entity. <br/></p>
</blockquote>
<p>Inherited from: <a href="../InformationEntity/">InformationEntity</a></p>
<details>
<summary>Implementation Guidance</summary>
* These may include methods that directly produced the Information Entity, or upstream/accessory methods that indirectly support creation of the Information Entity - e.g. methods used to produce data that was interpreted as evidence to generate a Statement of knowledge.<br/>
* This field captures terms representing specific INSTANCES of methods applied, vs the 'supportingMethodTypes' attribute which captures TYPES of methods used.<br/>
</details>
<p><a href="../supportingMethodTypes/">supportingMethodTypes</a> --&gt; <a href="../Coding/">Coding</a> [*] <br/></p>
<blockquote>
<p>Types of methodological approaches that were executed to directly or indirectly support creation of the Information Entity. <br/></p>
</blockquote>
<p>Inherited from: <a href="../InformationEntity/">InformationEntity</a></p>
<details>
<summary>Implementation Guidance</summary>
* This field captures terms representing TYPES of methods applied, vs the 'specifiedBy' or'supportingMethods' attributes which capture specific INSTANCES of methods used. These may include types of methods that directly produced the Information Entity, or upstream/accessory methods that indirectly support creation of the Information Entity<br/>
* Implementers should define a relevant source or set of method type codes/terms to use here, based on the needs of the domain or application.'<br/>
</details>
<p><a href="../derivedFrom/">derivedFrom</a> --&gt; <a href="../InformationEntity/">InformationEntity</a> [*] <br/></p>
<blockquote>
<p>Another Information Entity from which this Information Entity is derived, in whole or in part. <br/></p>
</blockquote>
<p>Inherited from: <a href="../InformationEntity/">InformationEntity</a></p>
<p><a href="../reportedIn/">reportedIn</a> --&gt; <a href="../String/">String</a> [*] <br/></p>
<blockquote>
<p>A document in which the Information Entity is reported. <br/></p>
</blockquote>
<p>Inherited from: <a href="../InformationEntity/">InformationEntity</a></p>
<details>
<summary>Implementation Guidance</summary>
* This attribute is used specifically to reference documents/publications where the Information Entity is expressed or reported.  For a Statement, this might be a publication where the authors express the statement in text.  For a Data Item, this might be a publication with a table or figure that reports the value of the  data.<br/>
</details>
<p><a href="../informationQuality/">informationQuality</a> --&gt; <a href="../Coding/">Coding</a> [0..1] <br/></p>
<blockquote>
<p>A qualitative term indicating the scientific rigor or reliability with which the information was generated/collected. <br/></p>
</blockquote>
<p>Inherited from: <a href="../InformationEntity/">InformationEntity</a></p>
<details>
<summary>Implementation Guidance</summary>
* This is typically based on the quality of design and execution of the study or curation activity that generated it (e.g. were relevant controls assessed to show instruments were working, were all samples taken care of and handled identically, are methods sound and well documented, etc.).<br/>
* The quality of information is intrinsic to the information itself, and not to a particular application of the information (e.g. as evidence for making an Assertion)<br/>
* The quality of information is one factor that goes into the confidence we have in the information''s veracity (i.e. that it is an accurate reflection of reality it intends to measure or describe). Other factors informing confidence may include who did it (we may just not trust some Agents), when (if data created 500 years ago, we may have less confidence in it), and for Assertions, the relevance and abundance of supporting evidence.<br/>
* Implementers should define a relevant source of codes or terms to use here, based on the needs of the domain or application.'<br/>
</details>
<p><a href="../recordMetadata/">recordMetadata</a> --&gt; <a href="../RecordMetadata/">RecordMetadata</a> [0..1] <br/></p>
<blockquote>
<p>Provenance metadata about a specific concrete record of information as encoded/serialized in a particular data set or object (as opposed to provenance about the abstract information content the encoding carries). <br/></p>
</blockquote>
<p>Inherited from: <a href="../InformationEntity/">InformationEntity</a></p>
<details>
<summary>Implementation Guidance</summary>
* This attribute holds a structured RecordMetadata objects, which can be used to capture when, how, and by whom a record serialization was generated or modified; what upstream resources it was derived or retrieved from; and record-level administrative information such as versioning and lifecycle status.<br/>
</details>
<p><a href="../id/">id</a> --&gt; <a href="../String/">String</a> [1] <br/></p>
<blockquote>
<p>The 'logical' identifier of the entity in the system of record, e.g. a UUID.  This 'id' is unique within a given system, but may or may not be globally unique outside the system. It is used within a system to reference one object from another. <br/></p>
</blockquote>
<p>Inherited from: <a href="../Entity/">Entity</a></p>
<details>
<summary>Implementation Guidance</summary>
* Note that it is common for implementers to create their own internal logical ids - typically a serially or randomly generated value like a UUID that is assigned to the data object as it is created in a system. But an implementer may choose to reuse an existing, globally unique id from an external system or authority for this purpose (e.g. an HGNC id for a Gene object) - as long as it is unique within the implementing system, and can be used to reference the identified object in this context.<br/>
</details>
<p><a href="../identifiers/">identifiers</a> --&gt; <a href="../String/">String</a> [*] <br/></p>
<blockquote>
<p>A globally-unique 'business' identifier or accession number for the real-world entity represented by a data object. These are typically assigned by an external system or authority, and used to connect entities and share content across different systems. <br/></p>
</blockquote>
<p>Inherited from: <a href="../Entity/">Entity</a></p>
<details>
<summary>Implementation Guidance</summary>
* Preferred values for this attribute are CURIEs or URIs - so the system that provisioned the identifier is clear.<br/>
* A given real world entity - e.g. a genetic variant - may have many business identifiers defined by different systems, which can be captured in the "identifiers" property to indicate that they represent the same thing.<br/>
</details>
<p><a href="../type/">type</a> --&gt; <a href="../String/">String</a> [1] <br/></p>
<blockquote>
<p>The name of the class that is instantiated by a data object representing the Entity. <br/></p>
</blockquote>
<p>Inherited from: <a href="../Entity/">Entity</a></p>
<details>
<summary>Implementation Guidance</summary>
* MUST be the label of a concrete class from the data model.<br/>
</details>
<p><a href="../label/">label</a> --&gt; <a href="../String/">String</a> [0..1] <br/></p>
<blockquote>
<p>A primary name for the Entity. <br/></p>
</blockquote>
<p>Inherited from: <a href="../Entity/">Entity</a></p>
<p><a href="../alternativeLabels/">alternativeLabels</a> --&gt; <a href="../String/">String</a> [*] <br/></p>
<blockquote>
<p>Alternative name(s) for the Entity. <br/></p>
</blockquote>
<p>Inherited from: <a href="../Entity/">Entity</a></p>
<p><a href="../description/">description</a> --&gt; <a href="../String/">String</a> [0..1] <br/></p>
<blockquote>
<p>A free text description of the Entity. <br/></p>
</blockquote>
<p>Inherited from: <a href="../Entity/">Entity</a></p>
<p><a href="../extensions/">extensions</a> --&gt; <a href="../Extension/">Extension</a> [*] <br/></p>
<blockquote>
<p>A list of extensions to the Entity, that allow for capture of information not directly supported by elements defined in the model. <br/></p>
</blockquote>
<p>Inherited from: <a href="../Entity/">Entity</a></p>
<details>
<summary>Implementation Guidance</summary>
* Extension objects have a key-value data structure that allows definition of custom fields in the data itself. Extensions are not expected to be natively understood, but may be used for pre-negotiated exchange of message attributes between systems.<br/>
</details>
<h2 id="usages">Usages</h2>
<table>
<thead>
<tr>
<th>used by</th>
<th>used in</th>
<th>type</th>
<th>used</th>
</tr>
</thead>
<tbody>
<tr>
<td><a href="./">StudyResult</a></td>
<td><a href="../componentResult/">componentResult</a></td>
<td>range</td>
<td><a href="./">StudyResult</a></td>
</tr>
</tbody>
</table>
<h2 id="identifier-and-mapping-information">Identifier and Mapping Information</h2>
<h3 id="schema-source">Schema Source</h3>
<ul>
<li>from schema: https://w3id.org/sepio-framework/sepio-linkml</li>
</ul>
<h2 id="mappings">Mappings</h2>
<table>
<thead>
<tr>
<th>Mapping Type</th>
<th>Mapped Value</th>
</tr>
</thead>
<tbody>
<tr>
<td>self</td>
<td>sepio_linkml:StudyResult</td>
</tr>
<tr>
<td>native</td>
<td>sepio_linkml:StudyResult</td>
</tr>
</tbody>
</table>
<h2 id="linkml-source">LinkML Source</h2>
<!-- TODO: investigate https://stackoverflow.com/questions/37606292/how-to-create-tabbed-code-blocks-in-mkdocs-or-sphinx -->
<h3 id="direct">Direct</h3>
<details>
<pre><code class="language-yaml">name: StudyResult
description: A collection of data items from a single study that pertain to a particular
  subject or experimental unit in the study, along with optional provenance information
  describing how these data items were generated.
title: Study Result
comments:
- '- StudyResults provide a useful way to capture a subset of items from a study dataset
  that are used as evidence in generating higher order knowledge assertions about
  the entity that is the focus of the study result. For example, a subset of allele
  frequency data items about a particular variant of interest in the complete gnomAD
  dataset can be reported in a StudyResult object, along with relevant provenance
  information, so they can be referenced as evidence for a higher-order assertion
  about the variant''s pathogenicity.'
- As an example, consider the comprehensive allele frequency dataset provided by gnomAD,
  which covers millions of variants. A curator looking to assess the pathogenicity
  of a particular variant might create a StudyResult object to capture a subset of
  data related to this focus allele, including  its count, frequency, homozygous frequency,
  along with metadata concerning the shared provenance or quality of this data. The
  StudyResult could then be references as a piece of evidence used to inform the focus
  alleles final pathogenicity classification.
- Note that the use of a Study Results is not reserved only for defining subsets of
  data from a larger HTP/clinical study data sets.  A StudyResult might be used to
  organize the data from a simple, small scale bench experiment - e.g. a western blot
  analysis of protein expression, or an in vitro binding assay focused on a single
  protein.  Even such small 'studies' can generate multiple data items and metadata,
  and a StudyResult object can be used to collect all or some of these data points
  pertinent to a particular focus into an organized structure.
from_schema: https://w3id.org/sepio-framework/sepio-linkml
status: Draft
is_a: InformationEntity
attributes:
  focus:
    name: focus
    description: A specific subject or experimental unit in a Study, that data in
      the StudyResult object is about - e.g. a particular variant in a population
      allele frequency dataset like ExAC or gnomAD.
    comments:
    - The 'focus' of a StudyResult is the what anchors selection of all data items
      and provenance information that it contains. This focus may be a single participant
      or subject of a study (e.g. one patient in a clinical study, or one allele in
      a population frequency analysis). It may be a study group defined as an experimental
      unit in the study (e.g. a specific treatment arm of a clinical study). Or it
      may be a variable defined as a unit of analysis in the study (e.g. 'exposure
      to nicotine' in an analysis correlating this variable with clinical outcomes).
    from_schema: https://w3id.org/sepio-model
    rank: 1000
    domain_of:
    - StudyResult
    range: string
    required: false
    multivalued: false
  dataItems:
    name: dataItems
    description: One or more data items that are included in the StudyResult because
      it pertains to the 'focus' of the result. This can be data that directly describes
      this 'focus' (e.g. the population frequency of an allele focus), or be metadata
      about how data about the 'focus' were generated (e.g the quality measures for
      the sequencing run used to determine this allele frequency).
    comments:
    - Any type of data item from a Study may be captured in a StudyResult - including
      raw experimental/observational data, derived calculations or quality measures,
      provenance information about how the data were generated in the study, or qualitative
      expert interpretations of the primary study data (e.g. the interpretation of
      a blood cholesterol value of 250 mg/dL as being 'high', or a Polyphen functional
      impact score of 0.115 as being 'deleterious').
    - The model specifies use of a key-value based 'DataItem' object to capture the
      meaning and value of each type of data item captured in a given StudyResult.
      But in practice, profiles for specific StudyResult types may choose to define
      one or more specializations of the generic 'dataItems' property as named attributes.
      This makes the data more succinct and parsable, and allows specific constraints
      to be applied and validated for different data items. For example, a PopulaitonAlleleFrequencyStudyResult
      profile may define a named 'focusAlleleFrequency' attribute that is required,
      and a named 'focusAlleleCount' attribute that is optional - both of which conceptually
      specialize the core-im 'dataItems' property. Under this approach, the core 'dataItems'
      acts as a placeholder to seed such specializations, but is not used directly
      in StudyResult profiles.
    from_schema: https://w3id.org/sepio-model
    status: Informative
    rank: 1000
    domain_of:
    - StudyResult
    range: DataItem
    required: false
    multivalued: true
  interpretation:
    name: interpretation
    description: The outcome of an interpretation of lower-level data item(s) in a
      StudyResult, that express some broader conclusion or insight that was made (e.g.
      that data indicates a result to be is high, normal, or low).
    comments:
    - This attribute allows basic interpretations / classifications to be included
      in a Study Result, such as binning a data item or items as being in or out of
      some normal range.
    from_schema: https://w3id.org/sepio-model
    status: Informative
    rank: 1000
    domain_of:
    - StudyResult
    range: Coding
    required: false
    multivalued: false
  sourceDataSet:
    name: sourceDataSet
    description: A larger DataSet from which the content of the StudyResult was derived.
    comments:
    - In most cases, a StudyResult will be generated using data from one source dataset
      - but it is possible multiple datasets related to a single study contain data
      about a particular focus that get collected into a single StudyResult instance.
    from_schema: https://w3id.org/sepio-model
    status: Draft
    rank: 1000
    domain_of:
    - StudyResult
    range: DataSet
    required: false
    multivalued: true
  componentResult:
    name: componentResult
    description: Another StudyResult comprised of data items about the same focus
      as its parent Result, but based on a more narrowly scoped analysis of the foundational
      data (e.g. an analysis based on data about a subset of the parent Results full
      study population).
    comments:
    - This attribute allows data creators to break down a StudyResult into finer-grained
      StudyResult instances with a narrower scope. For example, a StudyResult about
      the frequency of an allele in a global population of individuals can be broken
      down into separate 'component' Results about distinct subpopulations of individuals
      in the source data (e.g. frequency of the allele in asian vs african subpopulations,
      or male vs female subgroups).
    from_schema: https://w3id.org/sepio-model
    status: Draft
    rank: 1000
    domain_of:
    - StudyResult
    range: StudyResult
    required: false
    multivalued: true
  studyGroup:
    name: studyGroup
    description: A description of a specific group or population of subjects interrogated
      in the ResearchStudy that produced the data captured in the StudyResult.
    comments:
    - For example, in a StudyResult holding allele frequency data, this attribute
      points to a StudyGroup object that describes characteristics of the population
      that the frequency data was generated from (e.g. 'East Asian', 'Female').
    from_schema: https://w3id.org/sepio-model
    rank: 1000
    domain_of:
    - StudyResult
    range: StudyGroup
    required: false
    multivalued: false

</code></pre>
</details>
<h3 id="induced">Induced</h3>
<details>
<pre><code class="language-yaml">name: StudyResult
description: A collection of data items from a single study that pertain to a particular
  subject or experimental unit in the study, along with optional provenance information
  describing how these data items were generated.
title: Study Result
comments:
- '- StudyResults provide a useful way to capture a subset of items from a study dataset
  that are used as evidence in generating higher order knowledge assertions about
  the entity that is the focus of the study result. For example, a subset of allele
  frequency data items about a particular variant of interest in the complete gnomAD
  dataset can be reported in a StudyResult object, along with relevant provenance
  information, so they can be referenced as evidence for a higher-order assertion
  about the variant''s pathogenicity.'
- As an example, consider the comprehensive allele frequency dataset provided by gnomAD,
  which covers millions of variants. A curator looking to assess the pathogenicity
  of a particular variant might create a StudyResult object to capture a subset of
  data related to this focus allele, including  its count, frequency, homozygous frequency,
  along with metadata concerning the shared provenance or quality of this data. The
  StudyResult could then be references as a piece of evidence used to inform the focus
  alleles final pathogenicity classification.
- Note that the use of a Study Results is not reserved only for defining subsets of
  data from a larger HTP/clinical study data sets.  A StudyResult might be used to
  organize the data from a simple, small scale bench experiment - e.g. a western blot
  analysis of protein expression, or an in vitro binding assay focused on a single
  protein.  Even such small 'studies' can generate multiple data items and metadata,
  and a StudyResult object can be used to collect all or some of these data points
  pertinent to a particular focus into an organized structure.
from_schema: https://w3id.org/sepio-framework/sepio-linkml
status: Draft
is_a: InformationEntity
attributes:
  focus:
    name: focus
    description: A specific subject or experimental unit in a Study, that data in
      the StudyResult object is about - e.g. a particular variant in a population
      allele frequency dataset like ExAC or gnomAD.
    comments:
    - The 'focus' of a StudyResult is the what anchors selection of all data items
      and provenance information that it contains. This focus may be a single participant
      or subject of a study (e.g. one patient in a clinical study, or one allele in
      a population frequency analysis). It may be a study group defined as an experimental
      unit in the study (e.g. a specific treatment arm of a clinical study). Or it
      may be a variable defined as a unit of analysis in the study (e.g. 'exposure
      to nicotine' in an analysis correlating this variable with clinical outcomes).
    from_schema: https://w3id.org/sepio-model
    rank: 1000
    alias: focus
    owner: StudyResult
    domain_of:
    - StudyResult
    range: string
    required: false
    multivalued: false
  dataItems:
    name: dataItems
    description: One or more data items that are included in the StudyResult because
      it pertains to the 'focus' of the result. This can be data that directly describes
      this 'focus' (e.g. the population frequency of an allele focus), or be metadata
      about how data about the 'focus' were generated (e.g the quality measures for
      the sequencing run used to determine this allele frequency).
    comments:
    - Any type of data item from a Study may be captured in a StudyResult - including
      raw experimental/observational data, derived calculations or quality measures,
      provenance information about how the data were generated in the study, or qualitative
      expert interpretations of the primary study data (e.g. the interpretation of
      a blood cholesterol value of 250 mg/dL as being 'high', or a Polyphen functional
      impact score of 0.115 as being 'deleterious').
    - The model specifies use of a key-value based 'DataItem' object to capture the
      meaning and value of each type of data item captured in a given StudyResult.
      But in practice, profiles for specific StudyResult types may choose to define
      one or more specializations of the generic 'dataItems' property as named attributes.
      This makes the data more succinct and parsable, and allows specific constraints
      to be applied and validated for different data items. For example, a PopulaitonAlleleFrequencyStudyResult
      profile may define a named 'focusAlleleFrequency' attribute that is required,
      and a named 'focusAlleleCount' attribute that is optional - both of which conceptually
      specialize the core-im 'dataItems' property. Under this approach, the core 'dataItems'
      acts as a placeholder to seed such specializations, but is not used directly
      in StudyResult profiles.
    from_schema: https://w3id.org/sepio-model
    status: Informative
    rank: 1000
    alias: dataItems
    owner: StudyResult
    domain_of:
    - StudyResult
    range: DataItem
    required: false
    multivalued: true
  interpretation:
    name: interpretation
    description: The outcome of an interpretation of lower-level data item(s) in a
      StudyResult, that express some broader conclusion or insight that was made (e.g.
      that data indicates a result to be is high, normal, or low).
    comments:
    - This attribute allows basic interpretations / classifications to be included
      in a Study Result, such as binning a data item or items as being in or out of
      some normal range.
    from_schema: https://w3id.org/sepio-model
    status: Informative
    rank: 1000
    alias: interpretation
    owner: StudyResult
    domain_of:
    - StudyResult
    range: Coding
    required: false
    multivalued: false
  sourceDataSet:
    name: sourceDataSet
    description: A larger DataSet from which the content of the StudyResult was derived.
    comments:
    - In most cases, a StudyResult will be generated using data from one source dataset
      - but it is possible multiple datasets related to a single study contain data
      about a particular focus that get collected into a single StudyResult instance.
    from_schema: https://w3id.org/sepio-model
    status: Draft
    rank: 1000
    alias: sourceDataSet
    owner: StudyResult
    domain_of:
    - StudyResult
    range: DataSet
    required: false
    multivalued: true
  componentResult:
    name: componentResult
    description: Another StudyResult comprised of data items about the same focus
      as its parent Result, but based on a more narrowly scoped analysis of the foundational
      data (e.g. an analysis based on data about a subset of the parent Results full
      study population).
    comments:
    - This attribute allows data creators to break down a StudyResult into finer-grained
      StudyResult instances with a narrower scope. For example, a StudyResult about
      the frequency of an allele in a global population of individuals can be broken
      down into separate 'component' Results about distinct subpopulations of individuals
      in the source data (e.g. frequency of the allele in asian vs african subpopulations,
      or male vs female subgroups).
    from_schema: https://w3id.org/sepio-model
    status: Draft
    rank: 1000
    alias: componentResult
    owner: StudyResult
    domain_of:
    - StudyResult
    range: StudyResult
    required: false
    multivalued: true
  studyGroup:
    name: studyGroup
    description: A description of a specific group or population of subjects interrogated
      in the ResearchStudy that produced the data captured in the StudyResult.
    comments:
    - For example, in a StudyResult holding allele frequency data, this attribute
      points to a StudyGroup object that describes characteristics of the population
      that the frequency data was generated from (e.g. 'East Asian', 'Female').
    from_schema: https://w3id.org/sepio-model
    rank: 1000
    alias: studyGroup
    owner: StudyResult
    domain_of:
    - StudyResult
    range: StudyGroup
    required: false
    multivalued: false
  isAbout:
    name: isAbout
    description: An entity or concept in the world that the information entity describes/is
      about.
    comments:
    - e.g. In the context of a Statement object, this attribute may be used to indicate
      entities/concepts it is about, in lieu of providing a more precise description
      of what the Statement asserts to be true using subject, predicate, object, and
      qualifier properties. e.g. the Statement that "BRCA2 c.8023A&gt;G is pathogenic
      for Breast Cancer" might be annotated to be about the variant 'BRCA2 c.8023A&gt;G',
      and the disease 'Breast Cancer'.
    from_schema: https://w3id.org/sepio-model
    status: Informative
    rank: 1000
    alias: isAbout
    owner: StudyResult
    domain_of:
    - InformationEntity
    range: string
    required: false
    multivalued: true
  contributions:
    name: contributions
    description: Specific actions taken by an Agent toward the creation, modification,
      validation, or deprecation of an Information Entity.
    comments:
    - This attribute holds one or more Contribution objects, which provide structured
      descriptions of a contribution made to the Information Entity by a particular
      agent.
    from_schema: https://w3id.org/sepio-model
    status: Draft
    rank: 1000
    alias: contributions
    owner: StudyResult
    domain_of:
    - InformationEntity
    - RecordMetadata
    range: Contribution
    required: false
    multivalued: true
  dateAuthored:
    name: dateAuthored
    description: Indicates when the information content expressed in the Information
      Entity was generated.
    comments:
    - The term 'authored' as used in the model refers to the generation of 'information
      content' in the abstract  sense, as opposed to a concrete encoding of this information
      in a specific language or format. e.g. for a Statement, this attribute captures
      when the information content expressed in the Statement was first generated
      by an agent.  Information about when a particular concrete encoding of this
      information was created (e.g. as row in a table, or object in a json document)
      would live in a RecordMetadata object attached to the Information Entity).
    from_schema: https://w3id.org/sepio-model
    status: Draft
    rank: 1000
    alias: dateAuthored
    owner: StudyResult
    domain_of:
    - InformationEntity
    range: string
    required: false
    multivalued: false
  specifiedBy:
    name: specifiedBy
    description: A specification that describes all or part of the process that led
      to creation of the Information Entity.
    comments:
    - Examples - an experimental protocol or data analysis specification that describe
      how data were generated, or an evidence interpretation guideline that describes
      steps taken to interpret data in making a variant pathogenicity classification.
    - Note that this attribute captures specific *instances* of specifications/methods
      (e.g. the specific electron microscopy method described in https://doi.org/10.1002/
      cpz1.1045) - as opposed to reporting a *type* of method applied (e.g. "Transmission
      Electron Microscopy").
    from_schema: https://w3id.org/sepio-model
    status: Draft
    rank: 1000
    alias: specifiedBy
    owner: StudyResult
    domain_of:
    - InformationEntity
    - Activity
    range: string
    required: false
    multivalued: true
  supportingMethods:
    name: supportingMethods
    description: Specific methods that were executed to directly or indirectly support
      creation of the Information Entity.
    comments:
    - These may include methods that directly produced the Information Entity, or
      upstream/accessory methods that indirectly support creation of the Information
      Entity - e.g. methods used to produce data that was interpreted as evidence
      to generate a Statement of knowledge.
    - This field captures terms representing specific INSTANCES of methods applied,
      vs the 'supportingMethodTypes' attribute which captures TYPES of methods used.
    from_schema: https://w3id.org/sepio-model
    status: Informative
    rank: 1000
    alias: supportingMethods
    owner: StudyResult
    domain_of:
    - InformationEntity
    range: string
    required: false
    multivalued: true
  supportingMethodTypes:
    name: supportingMethodTypes
    description: Types of methodological approaches that were executed to directly
      or indirectly support creation of the Information Entity.
    comments:
    - This field captures terms representing TYPES of methods applied, vs the 'specifiedBy'
      or'supportingMethods' attributes which capture specific INSTANCES of methods
      used. These may include types of methods that directly produced the Information
      Entity, or upstream/accessory methods that indirectly support creation of the
      Information Entity
    - Implementers should define a relevant source or set of method type codes/terms
      to use here, based on the needs of the domain or application.'
    from_schema: https://w3id.org/sepio-model
    status: Informative
    rank: 1000
    alias: supportingMethodTypes
    owner: StudyResult
    domain_of:
    - InformationEntity
    range: Coding
    required: false
    multivalued: true
  derivedFrom:
    name: derivedFrom
    description: Another Information Entity from which this Information Entity is
      derived, in whole or in part.
    from_schema: https://w3id.org/sepio-model
    status: Draft
    rank: 1000
    alias: derivedFrom
    owner: StudyResult
    domain_of:
    - InformationEntity
    range: InformationEntity
    required: false
    multivalued: true
  reportedIn:
    name: reportedIn
    description: A document in which the Information Entity is reported.
    comments:
    - This attribute is used specifically to reference documents/publications where
      the Information Entity is expressed or reported.  For a Statement, this might
      be a publication where the authors express the statement in text.  For a Data
      Item, this might be a publication with a table or figure that reports the value
      of the  data.
    from_schema: https://w3id.org/sepio-model
    status: Draft
    rank: 1000
    alias: reportedIn
    owner: StudyResult
    domain_of:
    - InformationEntity
    range: string
    required: false
    multivalued: true
  informationQuality:
    name: informationQuality
    description: A qualitative term indicating the scientific rigor or reliability
      with which the information was generated/collected.
    comments:
    - This is typically based on the quality of design and execution of the study
      or curation activity that generated it (e.g. were relevant controls assessed
      to show instruments were working, were all samples taken care of and handled
      identically, are methods sound and well documented, etc.).
    - The quality of information is intrinsic to the information itself, and not to
      a particular application of the information (e.g. as evidence for making an
      Assertion)
    - The quality of information is one factor that goes into the confidence we have
      in the information''s veracity (i.e. that it is an accurate reflection of reality
      it intends to measure or describe). Other factors informing confidence may include
      who did it (we may just not trust some Agents), when (if data created 500 years
      ago, we may have less confidence in it), and for Assertions, the relevance and
      abundance of supporting evidence.
    - Implementers should define a relevant source of codes or terms to use here,
      based on the needs of the domain or application.'
    from_schema: https://w3id.org/sepio-model
    status: Informative
    rank: 1000
    alias: informationQuality
    owner: StudyResult
    domain_of:
    - InformationEntity
    range: Coding
    required: false
    multivalued: false
  recordMetadata:
    name: recordMetadata
    description: Provenance metadata about a specific concrete record of information
      as encoded/serialized in a particular data set or object (as opposed to provenance
      about the abstract information content the encoding carries).
    comments:
    - This attribute holds a structured RecordMetadata objects, which can be used
      to capture when, how, and by whom a record serialization was generated or modified;
      what upstream resources it was derived or retrieved from; and record-level administrative
      information such as versioning and lifecycle status.
    from_schema: https://w3id.org/sepio-model
    status: Draft
    rank: 1000
    alias: recordMetadata
    owner: StudyResult
    domain_of:
    - InformationEntity
    range: RecordMetadata
    required: false
    multivalued: false
  id:
    name: id
    description: The 'logical' identifier of the entity in the system of record, e.g.
      a UUID.  This 'id' is unique within a given system, but may or may not be globally
      unique outside the system. It is used within a system to reference one object
      from another.
    comments:
    - Note that it is common for implementers to create their own internal logical
      ids - typically a serially or randomly generated value like a UUID that is assigned
      to the data object as it is created in a system. But an implementer may choose
      to reuse an existing, globally unique id from an external system or authority
      for this purpose (e.g. an HGNC id for a Gene object) - as long as it is unique
      within the implementing system, and can be used to reference the identified
      object in this context.
    from_schema: https://w3id.org/sepio-model
    status: Draft
    rank: 1000
    alias: id
    owner: StudyResult
    domain_of:
    - Entity
    range: string
    required: true
    multivalued: false
  identifiers:
    name: identifiers
    description: A globally-unique 'business' identifier or accession number for the
      real-world entity represented by a data object. These are typically assigned
      by an external system or authority, and used to connect entities and share content
      across different systems.
    comments:
    - Preferred values for this attribute are CURIEs or URIs - so the system that
      provisioned the identifier is clear.
    - A given real world entity - e.g. a genetic variant - may have many business
      identifiers defined by different systems, which can be captured in the "identifiers"
      property to indicate that they represent the same thing.
    from_schema: https://w3id.org/sepio-model
    status: Informative
    rank: 1000
    alias: identifiers
    owner: StudyResult
    domain_of:
    - Entity
    range: string
    required: false
    multivalued: true
  type:
    name: type
    description: The name of the class that is instantiated by a data object representing
      the Entity.
    comments:
    - MUST be the label of a concrete class from the data model.
    from_schema: https://w3id.org/sepio-model
    status: Draft
    rank: 1000
    alias: type
    owner: StudyResult
    domain_of:
    - Entity
    range: string
    required: true
    multivalued: false
  label:
    name: label
    description: A primary name for the Entity.
    from_schema: https://w3id.org/sepio-model
    status: Draft
    rank: 1000
    alias: label
    owner: StudyResult
    domain_of:
    - Entity
    - Coding
    range: string
    required: false
    multivalued: false
  alternativeLabels:
    name: alternativeLabels
    description: Alternative name(s) for the Entity.
    from_schema: https://w3id.org/sepio-model
    status: Draft
    rank: 1000
    alias: alternativeLabels
    owner: StudyResult
    domain_of:
    - Entity
    range: string
    required: false
    multivalued: true
  description:
    name: description
    description: A free text description of the Entity.
    from_schema: https://w3id.org/sepio-model
    status: Draft
    rank: 1000
    alias: description
    owner: StudyResult
    domain_of:
    - Entity
    range: string
    required: false
    multivalued: false
  extensions:
    name: extensions
    description: A list of extensions to the Entity, that allow for capture of information
      not directly supported by elements defined in the model.
    comments:
    - Extension objects have a key-value data structure that allows definition of
      custom fields in the data itself. Extensions are not expected to be natively
      understood, but may be used for pre-negotiated exchange of message attributes
      between systems.
    from_schema: https://w3id.org/sepio-model
    status: Draft
    rank: 1000
    alias: extensions
    owner: StudyResult
    domain_of:
    - Entity
    range: Extension
    required: false
    multivalued: true

</code></pre>
</details>
</article>
</div>
<script>var tabs=__md_get("__tabs");if(Array.isArray(tabs))e:for(var set of document.querySelectorAll(".tabbed-set")){var tab,labels=set.querySelector(".tabbed-labels");for(tab of tabs)for(var label of labels.getElementsByTagName("label"))if(label.innerText.trim()===tab){var input=document.getElementById(label.htmlFor);input.checked=!0;continue e}}</script>
</div>
</main>
<footer class="md-footer">
<div class="md-footer-meta md-typeset">
<div class="md-footer-meta__inner md-grid">
<div class="md-copyright">
  
  
    Made with
    <a href="https://squidfunk.github.io/mkdocs-material/" rel="noopener" target="_blank">
      Material for MkDocs
    </a>
</div>
</div>
</div>
</footer>
</div>
<div class="md-dialog" data-md-component="dialog">
<div class="md-dialog__inner md-typeset"></div>
</div>
<script id="__config" type="application/json">{"base": "..", "features": ["content.tabs.link"], "search": "../assets/javascripts/workers/search.16e2a7d4.min.js", "translations": {"clipboard.copied": "Copied to clipboard", "clipboard.copy": "Copy to clipboard", "search.config.lang": "en", "search.config.pipeline": "trimmer, stopWordFilter", "search.config.separator": "[\\s\\-]+", "search.placeholder": "Search", "search.result.more.one": "1 more on this page", "search.result.more.other": "# more on this page", "search.result.none": "No matching documents", "search.result.one": "1 matching document", "search.result.other": "# matching documents", "search.result.placeholder": "Type to start searching", "search.result.term.missing": "Missing", "select.version.title": "Select version"}}</script>
<script src="../assets/javascripts/bundle.5a2dcb6a.min.js"></script>
<script type="module">import mermaid from "https://unpkg.com/mermaid@10.4.0/dist/mermaid.esm.min.mjs";
mermaid.initialize({});</script><script type="module">import mermaid from "https://unpkg.com/mermaid@10.9.0/dist/mermaid.esm.min.mjs";
mermaid.initialize({});</script></body>
</html>